### PR TITLE
Feat: add new workflow runners

### PR DESCRIPTION
**Pull request by aryan-rajoria**

### CODE REVIEW

**Review by prabhu — APPROVED** [review]

File: .github/workflows/build-meson.yml [hunk 1]
```diff
@@ -46,11 +46,7 @@ jobs:
         python blint-db/blint_db/cli.py -Z1
         echo $GITHUB_TOKEN | oras login ghcr.io -u $GITHUB_USERNAME --password-stdin
         cd blint-db
-        oras push ghcr.io/appthreat/blintdb-meson:v0.1 \
-          --config ../.oras/config.json:application/vnd.oras.config.v1+json \
-          --annotation-file ../.oras/annotations.json \
-          ./blint.db:application/vnd.appthreat.vdb.layer.v1+tar \
-        
+        echo "$GITHUB_USERNAME;;$GITHUB_TOKEN" | python ./.oras/orasclient.py
```
Comment (prabhu) [inline .github/workflows/build-meson.yml]: Don't print the tokens on the workflow. You can get them using `os.getenv`

File: .github/workflows/build-meson.yml.almalinux [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    runs-on: ['self-hosted', 'almalinux']
+    permissions:
+      contents: write
+      packages: write
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: AppThreat/blint-db
+        path: blint-db
+    - name: Set up Python
```
Comment (prabhu) [inline .github/workflows/build-meson.yml.almalinux]: setup-python is not required for self hosted agents. 3.12 is already installed.

File: .github/workflows/build-meson.yml.almalinux [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      with:
+        python-version: '3.12'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine build
+        cd blint-db && python -m pip install .
+    - name: Install apt package
+      run: |
+        sudo dnf clean all && sudo dnf update
```
Comment (prabhu) [inline .github/workflows/build-meson.yml.almalinux]: arm64 agent uses ubuntu.

File: .github/workflows/build-meson.yml.almalinux [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - name: Build and upload db
+      run: |
+        mkdir -p temp
+        python blint-db/blint_db/cli.py -Z1
+        echo $GITHUB_TOKEN | oras login ghcr.io -u $GITHUB_USERNAME --password-stdin
+        cd blint-db
+        oras push ghcr.io/appthreat/blintdb-meson:v0.1 \
+          --config ../.oras/config.json:application/vnd.oras.config.v1+json \
+          --annotation-file ../.oras/annotations.json \
+          ./blint.db:application/vnd.appthreat.vdb.layer.v1+tar \
```
Comment (prabhu) [inline .github/workflows/build-meson.yml.almalinux]: `vnd.appthreat.blintdb`

File: .oras/orasclient.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+input_value = input().split(";;")
+
+client.login(password=input_value[1], username=input_value[0])
+
+client.push(
+    name="ghcr.io/appthreat/blintdb-meson:v0.1",
+    files=[
+        ("./.oras/config.json", "application/vnd.oras.config.v1+json"),
+        ("./.oras/annotations.json", "application/vnd.oras.annotation.v1+json"),
+        ("./blint.db", "application/vnd.appthreat.vdb.layer.v1+tar"),
```
Comment (prabhu) [inline .oras/orasclient.py]: `vnd.appthreat.blintdb`

File: .github/workflows/build-meson.yml (hunk 1 above)
Comment (aryan-rajoria) [reply .github/workflows/build-meson.yml]: what would be a good way to login? as oras does not support `arm64` also some of the packages required for building projects are difficult to find in almalinux.

File: .github/workflows/build-meson.yml.almalinux (hunk 3 above)
Comment (aryan-rajoria) [reply .github/workflows/build-meson.yml.almalinux]: Yes would be removing this file

**Pull request merged**